app/src/main.ts

```diff
@@ -344,3 +352,35 @@ app.on('login', (event, webContents, request, authInfo, callback) => {
     createLoginWindow(callback);
   }
 });
+
+app.on(
+  'accessibility-support-changed',
+  (event: Event, accessibilitySupportEnabled: boolean) => {
+    log.debug('app.accessibility-support-changed', {
+      event,
+      accessibilitySupportEnabled,
+    });
+  },
+);
+
+app.on(
+  'activity-was-continued',
+  (event: Event, type: string, userInfo: any) => {
+    log.debug('app.activity-was-continued', { event, type, userInfo });
+  },
+);
+
+app.on('browser-window-blur', (event: Event, window: BrowserWindow) => {
+  log.debug('app.browser-window-blur', { event, window });
+});
+
+app.on('browser-window-created', (event: Event, window: BrowserWindow) => {
+  log.debug('app.browser-window-created', { event, window });
+  if (setupWindow !== undefined) {
+    setupWindow(window);
+  }
+});
+
+app.on('browser-window-focus', (event: Event, window: BrowserWindow) => {
+  log.debug('app.browser-window-focus', { event, window });
+});
```

Comment:
Why subscribe to these events (except `'browser-window-created'`) and not do anything with them?